client/my-sites/menus/menu-editable-item.jsx

```diff
@@ -296,28 +304,35 @@ var MenuEditableItem = React.createClass( {
 		return [
 			{
 				key: 'remove',
-				className: 'noticon noticon-trash',
+        			title: this.translate( 'Delete' , { textOnly: true } ),
```

Comment:
I think we can get rid of these `title` properties, since they're mostly redundant in UX terms.